var/spack/repos/builtin/packages/py-metpy/package.py

```diff
@@ -30,7 +30,8 @@ class PyMetpy(PythonPackage):
     depends_on("py-matplotlib@2.1.0:", type=("build", "run"))
     depends_on("py-numpy@1.16.0:", type=("build", "run"))
     depends_on("py-pandas@0.24.0:", type=("build", "run"))
-    depends_on("py-pint@0.10.1:", type=("build", "run"))
+    # Unable to Find "pint.unit" -- Module Not Found Error with py-pint@0.20:
+    depends_on("py-pint@0.10.1:0.19", type=("build", "run"))
```

Comment:
The other way to deal with this would be to update the version of metpy to something less...ancient. 1.6.2 is the latest and should work with current versions of things.

Reply:
Agreed ... there's no maintainer for these packages and I only have marginal interest in them (just need them as one dependency for another package). Can we go with what is here and defer an update of all these packages to a later time?

Reply:
That's fine.

Reply:
Thanks! Appreciate it